Assets/Scripts/HPPickupInteract.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+public class HPPickupInteract : MonoBehaviour
+{
+    // Private Variables \\
+    [SerializeField]
+    private int hpPickupVal { get; set; } = 100;
```

Comment:
Having `{ get; set; }` here is unnecessary because it doesn't change how `hpPickupVal` can be used in the script.

Reply:
Also, making `hpPickupVal` a property prevents it from being visible in the Inspector.

Reply:
I figured a future script might want to change the heal value of individual health packs, but I also figure that can be implemented then.